copper-server-impl/src/main/java/com/pogeyan/cmis/impl/cacheProvider/GoogleGuiceCacheProviderImpl.java

```diff
@@ -69,4 +69,9 @@ public void remove(String repositoryId, String key) {
 			typeCacheMap.invalidate(key);
 		}
 	}
+
+	@Override
+	public void removeAll(String repositoryId) {
+		repo.remove(repositoryId);
```

Comment:
Need a check if(repo.containsKey(repositoryId))?? There might be a situation in which if KEY is not found then it will throw KeyNotFoundException!!